src/lib/symbol_overrides.cc

```diff
@@ -31,6 +31,19 @@ __attribute__((weak)) void *reallocarray(void *ptr, size_t nmemb,
 __attribute__((weak)) void *pvalloc(size_t size) NOEXCEPT;
 // NOLINTNEXTLINE cert-dcl51-cpp
 __attribute__((weak)) int __libc_allocate_rtsig(int high) NOEXCEPT;
+
+// sized free functions (C23, not yet available in glibc)
```

Comment:
New ones ! Thanks for keeping an eye out for these. How do we find / refresh this list of APIs ?